sdk/search/search-documents/test/internal/serialization.spec.ts

```diff
@@ -16,7 +16,29 @@ describe("serialization.serialize", () => {
   it("circular", () => {
     const circluarInput: any = { a: null };
```

Comment:
whoops there was a typo here from before 😅

```suggestion
    const circularInput: any = { a: null };
```